src/Streamly/Internal/System/Process.hs

```diff
@@ -137,6 +138,7 @@ import qualified Streamly.Internal.FileSystem.Handle
     as Handle (readChunks, putChunks)
 import qualified Streamly.Unicode.Stream as Unicode
 import qualified Streamly.Internal.Unicode.Stream as Unicode (lines)
+import Data.Kind (Type)
```

Comment:
Do we want to make this a must?
We can add a no warning in the cabal file instead. This is tedious to write for every function.